docs/docsite/rst/porting_guides/porting_guide_2.9.rst

```diff
@@ -68,6 +68,7 @@ Noteworthy module changes
 
 * `vmware_dvswitch <vmware_dvswitch_module>` accepts `folder` parameter to place dvswitch in user defined folder. This option makes `datacenter` as an optional parameter.
 * `vmware_datastore_cluster <vmware_datastore_cluster_module>` accepts `folder` parameter to place datastore cluster in user defined folder. This option makes `datacenter` as an optional parameter.
+* `mysql_db <mysql_db_module>` now supports multiple databases in dump operation.
```

Comment:
Why do you add a porting guide entry for this? Nobody needs to adjust their playbooks for this change to still work (without errors, warnings or future problems) as far as I can see.

Reply:
Fixed